lib/tasks/import.rake

```diff
@@ -1,10 +1,12 @@
-# frozen_string_literal: true
+ # frozen_string_literal: true
```

Comment:
This line has crept in after being removed in https://github.com/alphagov/content-publisher/pull/1769

Reply:
Removed this line by rebasing against master